Source/SeaInk.Core/Entities/Tables/ExcelTable.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using ClosedXML.Excel;
+using SeaInk.Core.Models.Tables;
+
+namespace SeaInk.Core.Entities.Tables
+{
+    public class ExcelTable : ITable
+    {
+        private XLWorkbook _workbook = null!;
+        private string _pathToFile;
+
+
+        public int SheetCount => _workbook.Worksheets.Count;
+        
+        public int ColumnCount(TableIndex index)
+        {
+            return 0;
+        }
+
+        public int RowCount(TableIndex index)
+        {
+            return 0;
+        }
+
+        public void CreateSheet(TableIndex index)
+        {
+            _workbook.Worksheets.Add(index.SheetName);
+            Save();
+        }
+
+        public void DeleteSheet(TableIndex sheet)
+        {
+            _workbook.Worksheets.Delete(sheet.SheetName);
+
+            var oldPath = _pathToFile;
+            _pathToFile = SeparatePath(_pathToFile).path;
+            _pathToFile += "t.xlsx";
+            
+            Save();
+            File.Delete(oldPath);
+            
+            Rename(SeparatePath(oldPath).name);
+
+        }
+
+        public void Load(string address)
+        {
+            _workbook = new XLWorkbook(address);
+            _pathToFile = address;
+            Save();
+        }
+
+        public string Create(string path)
+        {
+            _workbook = new XLWorkbook();
+            _workbook.AddWorksheet("Important Sheet");
+            
+            _pathToFile = path;
+            return path;
+        }
```

Comment:
Создание таблицы выглядит довольно странно, т.к. таблица не сохраняется сразу после создания в файл, то есть `Create` - довольно символический нейминг
```suggestion
        public string Create(string path)
        {
            _pathToFile = path;

            _workbook = new XLWorkbook();
            _workbook.AddWorksheet("Sheet1");
            _workbook.SaveAs(path);

            return path;
        }
```